pkgs/by-name/cy/cyclonedx-cli/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildDotnetModule
+, fetchFromGitHub
+}:
+
+buildDotnetModule rec {
+  pname = "cyclonedx-cli";
+  version = "0.25.0";
+
+  src = fetchFromGitHub {
+    owner = "CycloneDX";
+    repo = "${pname}";
+    rev = "v${version}";
```

Helps to avoid branch/tag confusion.

```suggestion
    rev = "refs/tags/v${version}";
```